src/Workspaces/CoreTest/Differencing/MatchTests.cs

```diff
@@ -1,87 +0,0 @@
-﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
-
-using System;
-using Roslyn.Utilities;
-using Xunit;
-
-namespace Microsoft.CodeAnalysis.Differencing.UnitTests
-{
-    public class MatchTests
```

Comment:
These tests were removed because they depended on being able to instantiate the low level tree comparison API with a non-SyntaxNode.  Keeping around all that generic code for this purpose was a major burden on the API.  The move in the API to just use SyntaxNode made keeping these tests around not possible.

As we have tons of test around the true diffing algorithm for C# and VB, i don't think these low level tests add a lot of value, and so i removed them.